cmd/daily-heal-ops.go

```diff
@@ -72,6 +111,14 @@ func getLocalBackgroundHealStatus() madmin.BgHealState {
 	}
 }
 
+func getLocalBgHealingOpsStatus() BgHealingOpsStatus {
+	return BgHealingOpsStatus{
+		LastActivity: globalHealListener.lastActivity,
+	}
+}
+
+var globalHealListener = &healListener{}
+
```

Comment:
Can the healing related code changes come in a separate PR ? @vadmeste

Reply:
> Can the healing related code changes come in a separate PR ? @vadmeste

The thing is that we do one global objects listing which feeds both healing & lifecycle. Healing is done each 30 days, but lifecycle should be done each 24 hours.. so it is not easy to send healing in a separate PR